packages/synchro-charts/src/components/sc-widget-grid/sc-grid-tooltip.tsx

```diff
@@ -49,9 +51,13 @@ export class ScGridTooltip {
     const thereIsSomeData = this.propertyPoint != null || this.alarmPoint != null;
     const color = this.breachedThreshold ? this.breachedThreshold.color : undefined;
     const displaysMoreThanTitle = thereIsSomeData && this.isEnabled;
+    const icon = this.breachedThreshold ? this.breachedThreshold.icon : undefined;
+    const label = this.breachedThreshold ? this.breachedThreshold.label : undefined;
+    const unit = this.unit || '';
+    const value = this.value || this.propertyPoint?.y;
 
     return (
-      <div class="tooltip-container">
+      <div class="tooltip-container" style={{ width: 'inherit', height: 'inherit' }}>
```

Comment:
why hardcode `style` here that is static? could we just alter the `tooltip-container` class?

Reply:
Done --- Remove.  To inherit the width of the outermost parent element before.